dashboard/http_server_head.py

```diff
@@ -16,11 +16,8 @@
 from ray._private.utils import get_or_create_event_loop
 from ray._raylet import GcsClient
 from ray.dashboard.dashboard_metrics import DashboardPrometheusMetrics
-
-# All third-party dependencies that are not included in the minimal Ray
-# installation must be included in this file. This allows us to determine if
-# the agent has the necessary dependencies to be started.
-from ray.dashboard.optional_deps import aiohttp, hdrs
+from ray._private.internal_third_party import aiohttp  # noqa: F401
```

Comment:
We already have `python/ray/_private/thirdparty/xxx`. What's the difference between that folder and this folder?